README.md

```diff
@@ -4,95 +4,138 @@
 [![Container image](https://quay.io/repository/coreos/coreos-installer/status)](https://quay.io/repository/coreos/coreos-installer)
 [![crates.io](https://img.shields.io/crates/v/coreos-installer.svg)](https://crates.io/crates/coreos-installer)
 
-`coreos-installer` is a simple installer for Fedora CoreOS (FCOS) and Red Hat Enterprise Linux CoreOS (RHCOS).
-It contains minimal logic to download, verify and write an OS image to a target disk.
-It can be invoked as a standalone binary or during bootup via a dracut module.
+coreos-installer is a program to assist with installing Fedora CoreOS
```

Comment:
Is this intentionally not super-ticked to differentiate the project from the binary?

Reply:
Yup.